lab-andrew/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "lab-andrew",
+  "version": "0.1.0",
+  "description": "Async test and proctice",
+  "main": "index.js",
+  "directories": {
+    "test": "mocha"
```

Comment:
This is incorrect, it should be "test": "test", this refers to the name of your test directory, "test": "mocha" refers to the test command you're specifying.